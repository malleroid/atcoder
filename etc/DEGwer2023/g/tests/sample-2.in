6
-0.322191 -0.852465 0
-0.463288 -0.553583 1
0.378710 -0.346882 1
-0.489727 0.488028 0
-0.731142 0.227066 1
0.254757 -0.899035 0
