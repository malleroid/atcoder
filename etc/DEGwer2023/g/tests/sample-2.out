22.950966056549
